app/src/main/java/com/x8bit/bitwarden/ui/platform/components/scaffold/BitwardenScaffold.kt

```diff
@@ -42,18 +49,28 @@ fun BitwardenScaffold(
     contentColor: Color = BitwardenTheme.colorScheme.text.primary,
     contentWindowInsets: WindowInsets = ScaffoldDefaults
         .contentWindowInsets
-        .exclude(WindowInsets.navigationBars),
+        .union(WindowInsets.displayCutout)
+        .exclude(WindowInsets.navigationBars.only(WindowInsetsSides.Vertical)),
     content: @Composable (PaddingValues) -> Unit,
 ) {
     Scaffold(
         modifier = Modifier
             .semantics { testTagsAsResourceId = true }
             .then(modifier),
-        topBar = topBar,
+        topBar = {
+            Column {
+                topBar()
+                utilityBar()
```

Comment:
Technically, each screen that actually needs this could do it themselves. Any thought on how we want to handle this